<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wget</class>
 <widget class="QMainWindow" name="wget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>315</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>315</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>315</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WGET APPLICATION</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_Logo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/image/mepsanlogo1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>100</y>
      <width>291</width>
      <height>108</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_IP">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Enter IP Address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_IP"/>
     </item>
     <item>
      <widget class="QLabel" name="label_File_Names">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>File Names</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_File_Names">
       <item>
        <property name="text">
         <string>/Logs/MertSerialUpdateLogger.txt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/Logs/logger.txt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/Logs/logger2.txt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/Logs/logger2_port1.txt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/Logs/logger2_port2.txt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/Logs/logger2_port3.txt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/Logs/logger_port1.txt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/Logs/logger_port2.txt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/Logs/logger_port3.txt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/Logs/logger_port4.txt</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_Download">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>98</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DOWNLOAD</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>212</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="checkBox_Select_All">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>0</y>
       <width>142</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Select All</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="checkBox_Select_File_Location">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>20</y>
       <width>142</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Select File Location</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="checkBox_Select_File">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>40</y>
       <width>142</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Select File</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
